### PR TITLE
readme: update wording of tidb-tools readme

### PR DESCRIPTION
**Pull request by CaitinChen**
@GregoryIan PTAL Thanks

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,25 +16,26 @@ make sync_diff_inspector # build sync_diff_inspector
...
-When build successfully, you can find the binary in bin directory.
+When tidb-tools are built successfully, you can find the binary in the `bin` directory.
 
 ## Tool list
 
-[importer](./importer)
+- [importer](./importer)
 
-A tool for generating and inserting datas to database which is compatible with MySQL protocol, like MySQL, TiDB.
+    A tool for generating and inserting data to the database which is compatible with the MySQL protocol, like MySQL and TiDB.
```
Comment (kennytm) [inline README.md]: ... inserting data to **<ins>any</ins>** database ...

File: README.md [hunk 2]
```diff
@@ -17,25 +16,26 @@ make sync_diff_inspector # build sync_diff_inspector
...
 
-[checker](./checker)
+- [checker](./checker)
 
-A tool for checking the compatibility of an existed MySQL database with TiDB.
+    A tool for checking the compatibility of an existed MySQL database with TiDB.
```
Comment (kennytm) [inline README.md]: ... an **<ins>existing</ins>** MySQL ...

**Review by kennytm — COMMENTED** [review]
Rest LGTM.

**Comment by CaitinChen** [comment]
@kennytm  Thanks for your comments.

**Comment by IANTHEREAL** [comment]
LGTM

**Pull request merged**